src/partyfy.js

```diff
@@ -63,24 +51,48 @@ async function partyfy(imageBuffer, options = defaultOptions) {
   }
 }
 
-// Returns a gifwrap/jimp compatible Bitmap object
-function asyncGetPixelBitmap(imageBuffer) {
-  return new Promise((resolve, reject) => {
-    try {
-      const { mime } = fileType(imageBuffer);
-      getPixels(imageBuffer, mime, (err, { data, shape }) => {
-        if (err) return reject(err);
+// Returns gifwrap/jimp compatible frames
+async function readFrames(imageBuffer, opts) {
+  const { mime } = fileType(imageBuffer);
+
+  switch (mime) {
+    case 'image/gif': {
+      const { frames } = await GifUtil.read(imageBuffer)
+      const frameCount = frames.length
 
-        if (shape.length === 4) {
-          return reject('animated gifs are not currently supported');
+      if (frameCount > 1) {
+        if (opts.frameDelay != defaultOptions.frameDelay) {
+          console.warn('Warning: frameDelay is currently ignored for animated gifs.\n')
         }
 
-        resolve({ width: shape[0], height: shape[1], data });
-      });
-    } catch (err) {
-      reject(err);
+        return frames
+      } else {
+        return colors.map(() => new GifFrame(new BitmapImage(frames[0]),
+          { delayCentisecs: msToCs(opts.frameDelay) }
+        ))
+      }
     }
-  });
+    case 'image/png': 
```

Comment:
can you add `case image/jpg` this will also support jpg with no changes, after that I'm good to go for now until we figure out the other stuff.

Reply:
Done!